using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -329,7 +337,7 @@ Identity.getUrlVariables(new AdobeCallback<String>() {
 {% endtab %}
 
 {% tab title="iOS" %}
-### getUrlVariables
+### getUrlVariables<a id="geturlvariables-ios"></a>
 
 _added in ACPIdentity v2.1.0_
```

Comment:
Let's also update this ACPCore v2.3.0

Reply:
I'm trying to figure out wording here so it's not confusing. 
Something like `added in ACPIdentity v2.1.0 (bundled in ACPCore v2.3.0)` ?
Or more simply `added in ACPIdentity v2.1.0/ACPCore v2.3.0`?

Reply:
I think `added in ACPCore v2.3.0 (ACPIdentity v2.1.0)` works 👍